Remove Avalanche Wallet references

laviniat1996 opened the pull request.

yulin-dong commented:
I will review when I can.

ashucoder9 commented on docs/build/dapp/smart-contracts/remix-deploy.md:
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
 ## Step 2: Funding Your C-Chain Address
 
-### **Using Avalanche Wallet**
+### **Using Core web**
```
```suggestion
### **Using Core Web**
```

laviniat1996 replied:
@ashucoder9 That's a valid suggestion, I was conflicted on this one too. We use 'Core web' in our support docs https://support.avax.network/en/?q=core+web . I thought I should keep the same pattern here. What do you think I should do?

ashucoder9 commented on docs/build/dapp/smart-contracts/remix-deploy.md:
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
-Wallet can be used on test and local networks, too.
+[tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake).
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too. 
```
```suggestion
The Core wallet can be used on test and local networks, and you can download it on [mobile](https://support.avax.network/en/articles/6115608-core-mobile-where-can-i-download-core-mobile-to-my-phone) too.
```

laviniat1996 replied:
fixed aee59acc5055056042b9fc36caa38ebce92421d5

ashucoder9 commented on docs/build/dapp/smart-contracts/remix-deploy.md:
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
+[tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake).
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too. 
+This wallet is available for 
```
```suggestion
```

ashucoder9 commented on docs/build/dapp/smart-contracts/remix-deploy.md:
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too. 
+This wallet is available for 
+[mobile](https://support.avax.network/en/articles/6115608-core-mobile-where-can-i-download-core-mobile-to-my-phone) too.
```
```suggestion
```

ashucoder9 commented on docs/build/subnet/deploy/fuji-testnet-subnet.md:
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
+   which you will [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account)
+   to Core extension.
+
+3. Connect Core extension to [Core web](https://core.app/), and move the test funds from the C-Chain to the P-Chain
```
```suggestion
3. Connect the Core extension to [Core Web](https://core.app/), and move the test funds from C-Chain to the P-Chain
```

laviniat1996 replied:
fixed eb9c4fb7554477079d0b256e5e263d1d91435443

ashucoder9 commented on docs/build/subnet/deploy/fuji-testnet-subnet.md:
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
-   side of the web wallet (find more details on
-   [this tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet)).
+2. **Export** your key via the `avalanche key export` command. The output is your private key,
+   which you will [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account)
```
```suggestion
   which will help you [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account) your account
```

laviniat1996 replied:
fixed aee59acc5055056042b9fc36caa38ebce92421d5

laviniat1996 replied:
fixed eb9c4fb7554477079d0b256e5e263d1d91435443

ashucoder9 commented on docs/build/subnet/deploy/fuji-testnet-subnet.md:
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
-   [this tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet)).
+2. **Export** your key via the `avalanche key export` command. The output is your private key,
+   which you will [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account)
+   to Core extension.
```
```suggestion
into the Core extension.
```

laviniat1996 replied:
fixed eb9c4fb7554477079d0b256e5e263d1d91435443

ashucoder9 commented on docs/build/subnet/deploy/fuji-testnet-subnet.md:
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
+   to Core extension.
+
+3. Connect Core extension to [Core web](https://core.app/), and move the test funds from the C-Chain to the P-Chain
+   by clicking Stake, then Cross-Chain Transfer ( find more details on
```
```suggestion
   by clicking Stake, then Cross-Chain Transfer (find more details on
```

laviniat1996 replied:
fixed eb9c4fb7554477079d0b256e5e263d1d91435443

ashucoder9 commented on docs/build/subnet/deploy/fuji-testnet-subnet.md:
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
+
+3. Connect Core extension to [Core web](https://core.app/), and move the test funds from the C-Chain to the P-Chain
+   by clicking Stake, then Cross-Chain Transfer ( find more details on
+   [this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake) ).
```
```suggestion
   [this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake)).
```

laviniat1996 replied:
fixed eb9c4fb7554477079d0b256e5e263d1d91435443

ashucoder9 commented on docs/build/subnet/deploy/mainnet-subnet.md:
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
+connected to [Core extension](https://join.core.app/extension).
+
+You can load the Ledger's C-Chain address in Core extension, or load in a different private key to 
+[Core extension](https://join.core.app/extension), then connect to Core web . 
```
```suggestion
[Core extension](https://join.core.app/extension), and then connect to Core Web. 
```

ashucoder9 commented on docs/build/subnet/deploy/mainnet-subnet.md:
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
+You can load the Ledger's C-Chain address in Core extension, or load in a different private key to 
+[Core extension](https://join.core.app/extension), then connect to Core web . 
+
+You can move test funds from the C-Chain to the P-Chain by clicking Stake on Core web, then Cross-Chain Transfer ( find more details on
```
```suggestion
You can move test funds from C-Chain to P-Chain by clicking Stake on Core Web, then Cross-Chain Transfer (find more details on
```

ashucoder9 commented on docs/build/subnet/deploy/mainnet-subnet.md:
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
+[Core extension](https://join.core.app/extension), then connect to Core web . 
+
+You can move test funds from the C-Chain to the P-Chain by clicking Stake on Core web, then Cross-Chain Transfer ( find more details on
+[this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake) ).
```
```suggestion
[this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake)).
```

ashucoder9 reviewed (changes requested):
Thank you @laviniat1996, suggested some edits 🙏🏽

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
````diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
 ```
 
-## Add Node as Validator on Fuji via the Web Wallet
+## Add Node as Validator on Fuji via Core web
````
```suggestion
## Add Node as Validator on Fuji via Core Web
```

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
 
 For adding the new node as a Validator on the Fuji testnet's Primary Network you can
-use the [Avalanche Web Wallet](https://wallet.avax.network).
+use the [Core web](https://core.app/) 
```
```suggestion
use the [Core Web](https://core.app/) 
```

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
-online or compiled and run locally. The Avalanche Web Wallet is a multi-faceted
-jewel and offers validation/delegation, cross-chain transfers, reward
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
```
```suggestion
Core Web is a free, all-in-one command center that gives users a more intuitive and comprehensive way to view,
```

laviniat1996 replied:
fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
-jewel and offers validation/delegation, cross-chain transfers, reward
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
+and use across the Avalanche network, Avalanche Subnets, and Ethereum. 
```
```suggestion
and use dApps across the Avalanche network, its various Subnets, and Ethereum. 
```

laviniat1996 replied:
fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
+and use across the Avalanche network, Avalanche Subnets, and Ethereum. 
+Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free, 
```
```suggestion
Core Web is optimized for use with the Core browser extension and Core mobile (available on both iOS & Android).
```

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
+and use across the Avalanche network, Avalanche Subnets, and Ethereum. 
+Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free, 
+personal wallet (self-custodial). Together, they are key components of the Core product 
```
```suggestion
Together, they are key components of the Core product 
```

laviniat1996 replied:
fixed https://support.avax.network/en/?q=core+mobile

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
 
-Check which network the wallet is connected to by looking at the top right of
-the screen. By default the Avalanche Web Wallet connects to Mainnet.
+By default, Core web and Core extension are connected to Mainnet. 
```
```suggestion
By default, Core Web and Core extension are connected to Mainnet. 
```

ashucoder9 commented on docs/nodes/run/third-party/aws-marketplace-one-click.md:
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
 
-<img src="/img/one-click-validator-node/network-fuji.png" alt="Network - Fuji"
-width="60%" />
+To add a node as a Validator, first select the Stake tab on Core web, 
```
```suggestion
To add a node as a Validator, first select the Stake tab on Core Web, 
```

ashucoder9 commented on docs/nodes/validate/add-a-validator.md:
```diff
@@ -46,9 +46,9 @@ when executing the AvalancheGo binary, as by default, the node will attempt to p
+## Add a Validator with Core extension
 
-First, we show you how to add your node as a validator by using [Avalanche Wallet](https://wallet.avax.network).
+First, we show you how to add your node as a validator by using [Core web](https://core.app).
```
```suggestion
First, we show you how to add your node as a validator by using [Core Web](https://core.app).
```

ashucoder9 commented on docs/nodes/validate/add-a-validator.md:
```diff
@@ -76,28 +76,30 @@ The response has your node’s ID:
 
-Open [the wallet](https://wallet.avax.network/), and go the `Earn` tab. Choose
-`Add Validator` under the `Validate` section.
+Connect [Core extension](https://join.core.app/extension) to [Core web](https://core.app), and go the 'Staking' tab. 
```
```suggestion
Connect [Core extension](https://join.core.app/extension) to [Core Web](https://core.app), and go to the 'Staking' tab. 
```

ashucoder9 commented on docs/nodes/validate/add-a-validator.md:
```diff
@@ -76,28 +76,30 @@ The response has your node’s ID:
-Open [the wallet](https://wallet.avax.network/), and go the `Earn` tab. Choose
-`Add Validator` under the `Validate` section.
+Connect [Core extension](https://join.core.app/extension) to [Core web](https://core.app), and go the 'Staking' tab. 
+Here, choose`Validate` from the menu.
```
```suggestion
Here, choose 'Validate` from the menu.
```

ashucoder9 commented on docs/reference/standards/cryptographic-primitives.md:
```diff
@@ -184,10 +184,14 @@ As an example we will sign the message "Through consensus to the stars"
 
-![Sign message](/img/sign-message.png)
+:::info
+A full guide on how to sign messages with Core web can be found 
```
```suggestion
A full guide on how to sign messages with Core Web can be found 
```

ashucoder9 reviewed (changes requested):
Suggestion: you may mark this PR as a draft @laviniat1996 until it's ready for a final review.

laviniat1996 commented on docs/build/subnet/deploy/mainnet-subnet.md:
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
-of the web wallet. See this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet-between-x-and-c-chain)
-for more instructions.
+exporting them from the C-Chain to the P-Chain using [Core web](https://core.app)
```
fixed 8033a0c4a32eae85925c0dc9eb6c4674ee616781

laviniat1996 commented:
@ashucoder9 The PR is ready now (except is not translated to spanish). I think I've addressed all of your comments. We still have to choose if we use capital letters for 'mobile' and 'web'.

ashucoder9 commented:
/gt docs/build/dapp/fuji-workflow.md i18n/es/docusaurus-plugin-content-docs/current/build/dapp/fuji-workflow.md spanish

ashucoder9 commented:
/gt docs/build/dapp/smart-contracts/remix-deploy.md i18n/es/docusaurus-plugin-content-docs/current/build/dapp/smart-contracts/remix-deploy.md spanish

ashucoder9 commented:
/gt docs/build/subnet/deploy/fuji-testnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md spanish

ashucoder9 commented:
/gt docs/nodes/validate/add-a-validator.md i18n/es/docusaurus-plugin-content-docs/current/nodes/validate/add-a-validator.md spanish

ashucoder9 commented:
/gt docs/build/subnet/deploy/fuji-testnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md spanish
